File: utilities/plot_omega_dwk.py
#!/usr/bin/env python

omega_dwk=[]
datafile=open('dwk_omega_dwk.out');
data=datafile.readlines()
num =len(data)

import numpy
returnMat = numpy.zeros((num,4))
index = 0
for line in data:
	line = line.strip()
	linelist = line.split()
	returnMat[index,:] = linelist[0:4]
	index +=1


import matplotlib.pyplot as plt
import matplotlib
from matplotlib.pyplot import rc, figure, axes, plot, xlabel, ylabel, title, \
	grid, savefig, show

omega_r =returnMat[:,0]
omega_i =returnMat[:,1]
dwk_r =returnMat[:,2]
dwk_i =returnMat[:,3]


font ={ 'size' : '18'}
rc('font',**font)
rc('font',**{'family':'sans-serif','sans-serif':['Computer Modern Roman']})

rc('text', usetex=True)
fig1=plt.figure(figsize=(8,8.5),)

rect = fig1.patch
rect.set_facecolor('white')
plt.subplot(2,1,1)

plt.plot(omega_r,dwk_r,'ro',markersize=6)
plt.xlabel(r'$\omega$')
plt.ylabel(r'$real(dwk)$')


ax=plt.subplot(2,1,2,axisbg='white')
plt.plot(omega_r,dwk_i,'ro',markersize=6)

plt.xlabel(r'$\omega$')
plt.ylabel(r'$imag(dwk)$')

plt.show()

File: utilities/plot_omega_dwk.m
function plot_omega_dwk(filename)

if(nargin<1)
    filename='omega_dwk.out';
end
clf;

data=load(filename);

romega=data(:,1);
iomega=data(:,2);

rdwk=data(:,3);
idwk=data(:,4);

 set(gcf,'Units','points','position',[100 500 1200 600],'Color',[1 1 1]);
% 
% % 
 hax=axes('Position',[0.08 0.15 0.4 0.75],'FontSize',24,'FontName','Latex'); 
 plot(romega,rdwk-0.064,'ro--');
 
 grid on;
 
%  ylim([0-0.12 0.1]);
% xlim([0 1.1]);
xlabel('$real(\omega)$');
ylabel('$real(\delta W_k)$');

text(0.4,0.11,'$dwk++:~\Lambda_0=0.28,\Delta \Lambda=0.1,\Delta E=0.3, E_c=0.25, \Delta r=0.001$','FontSize',24);



 hax=axes('Position',[0.56 0.15 0.4 0.75],'FontSize',24); 
  plot(romega,idwk,'ro--');

% ylim([0-0.12 0.15]);
% xlim([0 1.1]);
  grid on;
  xlabel('$real(\omega)$');
ylabel('$imag(\delta W_k)$');
  
 % myprint('dwk_omega_dwk');
 
end